src/constants.ts

```diff
@@ -1,7 +1,8 @@
 import { TMat2D } from './typedefs';
+// use this syntax so babel plugin see this import here
+import { version } from '../package.json';
 
-// TODO: consider using https://github.com/swiing/rollup-plugin-import-assertions so we can import json in node and have rollup build pass
-export { version as VERSION } from '../package.json';
+export const VERSION = version;
```

Comment:
a note about import assertion.
Looks like that the whole JSON as to be imported as a default import, or at least this is what was happening while i added the babel plugin to support import assertion.

Reply:
weird that the syntax must be like this. but babel...

Reply:
is just the plugin i found that is not great.
If i use the good one, to support assertion, i get all the package.json in the codebase.
While this is a plugin that swaps the statement with a const, but has been poorly written